docs/jet/design-docs/021-sql-rolling-upgrade.md

```diff
@@ -37,7 +37,7 @@ half are of version `v+1`. If the two groups of members with the same
 version have the same size, we choose the group with higher version; the
 reason is that members of `v+1` version will not be shut down. To
 mitigate this, the user can choose to temporarily increase the cluster
-size, the enterprise licence allows it.
```

Comment:
This is actually not a typo. Both are correct. You can read the whole explanation on [Grammarly](https://www.grammarly.com/blog/licence-license/)

Reply:
My bad, wasn't aware of this. Reverted the change.